### ISSUE TITLE
Rename jwt to authToken in config

### ISSUE THREAD

**Issue by CodingDoug** [libsql/kysely-libsql#1]
```
export interface LibsqlDialectConfig {
    client?: hrana.Client,
    url?: string,
    jwt?: string,
}
```

Could we rename `jwt` to `authToken` for parity with our other libraries?

**Comment by CodingDoug** [comment]
Pinging this issue, ran into it again today.

**Comment by honzasp** [comment]
Thanks for pointing out this issue, it is now fixed in version 0.2.0 :)
